code/ship/ship.h

```diff
@@ -1129,6 +1129,13 @@ class ship_info
 	float		max_shield_regen_per_second;	// Goober5000 - max percent/100 of shield energy regenerated per second
 	float		max_weapon_regen_per_second;	// Goober5000 - max percent/100 of weapon energy regenerated per second
 
+	float		shield_weap_amount;
+	float		shield_weap_efficiency;
+	float		shield_weap_speed;
+	float		weap_shield_amount;
+	float		weap_shield_efficiency;
+	float		weap_shield_speed;
```

Comment:
Could we get some comments here? Nothing complicated, just so its clear this is referring to the ets system, and what these values do, like the speeds are default 0.04, 0.04 of what?